_opensearch/rest-api/rank-eval.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Ranking evaluation
+parent: REST API reference
+nav_order: 82
+---
+
+# Ranking evaluation
+
+The [rank]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/rank/) eval endpoint allows you to evaluate the quality of ranked search results.
+
+## Path and HTTP methods
+
+```
+GET <index_name>/_rank_eval 
+POST <index_name>/_rank_eval
+```
+
+## URL parameters
+
+URL Parameters are optional.
+
+Parameter | Data Type | Description
+:--- | :---  | :---
+[`ignore_unavailable`](#ignore_unavailable) | boolean | Defaults to `false`. When set to `false` the response body will return an error if an index is closed or missing.
+[`allow_no_indices`](#allow_no_indices) | boolean | Defaults to `true`. When set to `false` the response body will return an error if a wildcard expression points to an indices that are closed or missing.
+[`expand_wildcards`](#expand_wildcards) | enum | Expand wildcard expressions for indicies that are open, closed, or both.
```

Comment:
I get that it is an enum, but for users' purposes I think it makes more sense to make this a string and enumerate the options with explanations.